lib/xmlrpc/server.rb

```diff
@@ -532,6 +532,49 @@ def http_write(body, status, header)
 end
 
 
+# Implements a XML-RPC server, which works with Rack
+class RackServer < BasicServer
```

Comment:
In general, Rack server means Puma, Passenger and so on.

```suggestion
class RackApplication < BasicServer
```